dsgrid/config/dimensions.py

```diff
@@ -20,6 +20,7 @@
     TimeDimensionType,
     RepresentativePeriodFormat,
     DatetimeFormat,
+    LeapDayAdjustmentType,
```

Comment:
Why wasn't this needed before? Maybe the Daily time models are in the wrong place?